TeamPage.html

```diff
@@ -1267,6 +1253,10 @@ <h6 id="contactID" class="h6 text-uppercase mb-4 font-weight-bold">Contact</h6>
             <p>
               <i class="fas fa-envelope mr-3" aria-label="Envelope Icon"></i><a aria-describedby="contactID"
                 href="mailto:gretchen.andrew@unlv.edu">gretchen.andrew@unlv.edu</a><br>
+              <i class="fas fa-phone mr-3" aria-label="Phone Icon"></i><a aria-describedby="contactID"
+                href="tel:7028951437">(702)-895-1437 (Phone)</a><br>
+              <i class="fas fa-print mr-3" aria-label="Fax Icon"></i><a aria-describedby="contactID"
+                href="tel:7028954064">(702)-895-4064 (Fax)</a><br>

```

Comment:
Was this added by accident or did Natalie request we re-add these to the website?

Reply:
OMG twas an accident! Apologies! I don't think I pulled down the new changes! Fixing it